detoul

```diff
@@ -117,8 +121,6 @@ save-spec() {
   cd - >/dev/null
 }
 
-((
```

Comment:
I don't think that's what you meant to do.

Reply:
yep :-) found it out last night. Will put those back in place.